tests/models/llava_next/test_modeling_llava_next.py

```diff
@@ -485,3 +490,54 @@ def test_small_model_integration_test_unk_token(self):
             self.processor.decode(output[0], skip_special_tokens=True),
             EXPECTED_DECODED_TEXT,
         )
+
+    @slow
+    @require_bitsandbytes
+    def test_small_model_integration_test_batch_different_resolutions(self):
```

Comment:
What I'd really like to see in a test to make sure the batching behaviour is as expected is comparing the outputs for a batch of two images, vs the logits for the images passed in individually

Reply:
done